### ISSUE TITLE
Unable to access email headers

### ISSUE THREAD

**Issue by jakeprime** [thoughtbot/griddler#205]
I am not able to see any headers for received emails. `@email.headers` returns `{}` and `@email.raw_headers` returns an empty string.

I am trying to get to the Date property. It is possible (e.g. if my app is temporarily down) for an email to be received at a different time to when it was sent, and if there are multiple they can be received out of order. I would like to have the sent time so I can have a correct timeline of emails.

I'm not sure if this is an issue with Griddler, or the Mandrill adapter I am using, I'm trying here first.

**Comment by bradpauly** [comment]
@jakeprime it's up to the adapter to make the headers available. I just took a quick look and it doesn't look like the Mandrill adapter uses them.

**Comment by jakeprime** [comment]
OK, I've opened this on griddler-mandrill: https://github.com/wingrunr21/griddler-mandrill/issues/16

**Comment by gabebw** [comment]
I'll wait on that PR to see if it's a problem with Griddler, griddler-mandrill, or both.

**Comment by wingrunr21** [comment]
I believe this is a duplicate of the issue raised in wingrunr21/griddler-mandrill/issues/12 and subsequently requires thoughtbot/griddler/pull/185 to be merged.

**Comment by gabebw** [comment]
#185 is now merged.
